mm2src/coins/qrc20.rs

```diff
@@ -1287,6 +1287,14 @@ impl MmCoin for Qrc20Coin {
         dex_fee_amount: DexFee,
         stage: FeeApproxStage,
     ) -> TradePreimageResult<TradeFee> {
+        if dex_fee_amount.zero_fee() {
+            return Ok(TradeFee {
+                coin: self.platform.clone(),
+                amount: MmNumber::default(),
+                paid_from_trading_vol: false,
+            });
+        }
+
```

Comment:
As I can see it is used in legacy
<img width="691" alt="image" src="https://github.com/user-attachments/assets/ddbc8a00-30e1-4e97-87f1-a2e15f61464e" />